.cargo-deny.toml

```diff
@@ -98,35 +98,38 @@ ignore = [
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
 # The lint level for crates which do not have a detectable license
-unlicensed = "allow"
+# unlicensed = "allow"
 # List of explicitly allowed licenses
 # See https://spdx.org/licenses/ for list of possible licenses
 # [possible values: any SPDX 3.11 short identifier (+ optional exception)].
 allow = [
     "MIT",
     "Apache-2.0",
     "Apache-2.0 WITH LLVM-exception",
+    "Zlib",
+    "BSD-3-Clause",
+    "ISC",
+    "Unicode-DFS-2016",
+    "LGPL-3.0"
```

Comment:
Are these values determined by our business? Is there a source for where guidance on allowable licenses come from (ie, Google doc, Slack message, email...?)

Reply:
That's a question that I also have

Reply:
Based on Wil's comment we should be good regarding these licenses